README.md

```diff
@@ -23,6 +23,13 @@ juju deploy ./prometheus-edge-hub_ubuntu-20.04-amd64.charm \
  --resource prometheus-edge-hub-image=facebookincubator/prometheus-edge-hub:1.1.0
 ```
 
+### Relating to Prometheus
+
+```bash
+juju deploy prometheus-k8s --channel edge
```

Comment:
channel=edge I think

and, actually, it’d be best to link to juju.is/docs/lma2, specifically to the MicroK8s tutorial :-)

Reply:
Both works, but yeah, I also suggest linking to the docs as it may (will) change over time.

Reply:
I added the ref and it works fine with `--channel edge`